capabilities/navigation/nearby-poi/maps/google-apis-maps.suma

```diff
@@ -0,0 +1,108 @@
+profile = "navigation/nearby-poi@1.0"
+provider = "google-apis-maps"
+
+map NearbyPoi {
```

Comment:
Is this intentional? Shouldn't this be `NearbyPoint`?

Reply:
Same goes to profile and usecases. 🤷🏻‍♂️

Reply:
POI => Point Of Interest 🤷

Reply:
Oh, ok 😄